### ISSUE TITLE
Can't click on search results on Android

### ISSUE THREAD

**Issue by edonadei** [jaing/react-native-places-input#2]
Hello, just wanted to warn you with an issue on Android emulator (Google Pixel 3), we can't click on the searchables results, so it won't trigger the onSelect.

My exemple on Expo:

```javascript
import React from "react";
import PlacesInput from "react-native-places-input";

export default function NewTerminal() {
  return (
      <PlacesInput
        googleApiKey={"MY_API_KEY"}
        onSelect={place => console.log(place)}
      />
  );
}
```

[Screen of what happens](https://i.imgur.com/P01Qv0o.png)

**Comment by jaing** [comment]
Fixed in 1.0.7. Issue was with Android not able to trigger on press in scroll view with opened keyboard and absolute position.

**Comment by edonadei** [comment]
Hello again, it looks like the exact same issue is still on with the 1.0.8 version of your package, 

[Here is now a video of what happens](https://i.imgur.com/98bmBfX.gifv)

**Comment by jaing** [comment]
Hm - you sure mate?
I tested both android and iphone:

https://drive.google.com/open?id=1WjEsQya07xjZEfsRGjffbAXOrShjd4jP

and its working fine. Can you provide more details about your case?

**Comment by egecavusoglu** [comment]
I am having a similar issue with absolute positioning on react-native-maps.The map collects the touch events rather than the list.

**Comment by shawnmitchell** [comment]
I've got the same Android issue with 1.1.2. Can't interact with the results list.

**Comment by jaing** [comment]
Is the map in a scroll view?

**Comment by shawnmitchell** [comment]
The list is in an <Animated.View> but not a scroll view and in fact there
is no map, just a straight address lookup for a bottom sheet form.

**Comment by jaing** [comment]
Can you give me your full example with styles included?

**Comment by shawnmitchell** [comment]
Okay, I looked into this a bit more. I was doing some `zIndex` stuff with my bottom sheet that was confusing the dropdown list. I set up a clean test view with just 

```
<View>
    <PlacesInput
       ...
    />
    <Button
      ...
    />
</View>
```
and found that while the dropdown list worked fine in iOS, when I tried to select the first item in the list on Android, the button behind the list was being triggered. I added `stylesContainer={{shadowOpacity: 0, zIndex: 2}}` to `PlacesInput` and the top item in the list became selectable. Hopefully this is helpful.
